### ISSUE TITLE
Bruno Environment Export

### ISSUE THREAD

**Issue by ShahbazyanS** [usebruno/bruno#2017]
I imported the environment from Postman, then made some updates, but was unable to export the modified environment from Bruno.
I think this needs to be introduced in Bruno.

**Comment by PushpenderSaini0** [comment]
Bruno environment are saved under `environments` directory in a human readable format, are you looking for a feature to export them in some specific format ?

**Comment by ShahbazyanS** [comment]
I think the ability to **export** the **environment** is a must, what about the format, I think it should be **JSON**.

**Comment by chenjianfei1989** [comment]
I want this feature to facilitate two-way migration with Postman. Thank you. @sanjai0py

**Comment by tivo-csingh** [comment]
Are we aware when can we have export (Bruno) environment option available in Bruno which can be shared to others so that they can import it easily.

**Comment by paulverbeke** [comment]
> I want this feature to facilitate two-way migration with Postman. Thank you. [@sanjai0py](https://github.com/sanjai0py)

I also would love two-way migration with Postman since my team don't use Bruno

**Comment by anusree-bruno** [comment]
Import and Export is supported in environments. Closing this issue.

**Comment by paulverbeke** [comment]
yes you can export an Env but not in Postman format. Or at least I didn't find a way to select a format other than Bruno. It prevents teams to collaborate when both Postman and Bruno are used
